run_performance_tests_against_local.sh

```diff
@@ -15,7 +15,7 @@ mv src/main/scala/uk/gov/dhsc/htbhf/RunGatlingTests.scala src/main/scala/uk/gov/
 
 export BIN_DIR="./bin"
 export GATLING_FOLDER_NAME="gatling-charts-highcharts-bundle-3.0.1.1"
-export GATLING_URL="http://localhost:8080"
+export GATLING_URL="https://repo1.maven.org/maven2/io/gatling/highcharts/gatling-charts-highcharts-bundle/3.0.1.1/gatling-charts-highcharts-bundle-3.0.1.1-bundle.zip"
```

Comment:
Why is this change needed?

Reply:
The GATLING_URL variable is where gatling should be downloaded from. I can't see how this ever worked.